libs/datasets/timeseries.py

```diff
@@ -147,16 +144,12 @@ def get_records_for_state(self, state) -> List[dict]:
         pd_data = self.get_subset(AggregationLevel.STATE, state=state).data
         return pd_data.where(pd.notnull(pd_data), None).to_dict(orient="records")
 
-    def get_data(
-        self, country=None, state=None, county=None, fips=None, states=None
-    ) -> pd.DataFrame:
+    def get_data(self, country=None, state=None, fips=None, states=None) -> pd.DataFrame:
```

Comment:
can you make this change on the latest values dataset as well? (not sure if it has county or not)

Reply:
Done